spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -172,39 +174,25 @@ def __del__(self):
             self.restart_thread.wait()
 
     # ----- Private methods ---------------------------------------------------
-    def _before_prompt_is_ready(self):
```

Comment:
because sig_prompt_ready and sig_remote_execute are replaced, so the only thing that this function does is call _show_loading_page, so call to this function can be replaced by _show_loading_page